dialogflow-cx/tests/test_basic_webhook_snippet.py

```diff
@@ -0,0 +1,69 @@
+# Copyright 2022 Google LLC
```

Comment:
I don't think this snippet should live here. I can see that it's self-testing, but could you tell me why it's better to test here rather than in a separate test file?

Having the `basic_webhook_sample` included in this PR would be helpful for understanding how the code all comes together.